components/contract-tabs/code/ContractCode.tsx

```diff
@@ -67,9 +67,8 @@ const INSTALL_COMMANDS = {
 };
 
 const CREATE_APP_COMMANDS = {
-  evm: "npx thirdweb@latest create --app",
-  solana:
-    "npx thirdweb@latest create --template next-typescript-solana-starter",
+  evm: "npx thirdweb@latest create app --evm",
```

Comment:
this should be --app for both
```suggestion
  evm: "npx thirdweb@latest create --app --evm",
```